docs/sample_config.yaml

```diff
@@ -978,6 +978,8 @@ metrics_flags:
 # Whether or not to report anonymized homeserver usage statistics.
 # report_stats: true|false
 
+# The endpoint where to report the anonymized homeserver usage statistics to. Defaults to matrix.org
```

Comment:
this file is auto-generated: you'll need to change the source in `synapse/config/metrics.py`

Reply:
Can't quite figure out how to test it, as running `./scripts-dev/generate_sample_config` seems to ignore any changes to `synapse/config/metrics.py` no matter what. Adding a simple print statement in its `generate_config_section` doesn't even print anything, indicating it isn't been called.

Reply:
sounds like you've got a `synapse/config/metrics.py` installed in your virtualenv which is taking precedence over the one in your working copy.

Reply:
That was it, thanks!